plugins/services/src/js/filters/ServiceAttributeIsFilter.js

```diff
@@ -2,16 +2,6 @@ import DSLFilterTypes from "#SRC/js/constants/DSLFilterTypes";
 import DSLFilter from "#SRC/js/structs/DSLFilter";
 import * as ServiceStatus from "../constants/ServiceStatus";
 
-const LABEL = "is";
```

Comment:
Why remove this constant?

Reply:
good question. i felt like it is really verbose and is not going to change anytime - thus i inlined it to make clearer that it can really be considered hardcoded. but i have no strong feelings here and would change it back if you think that's better.

Reply:
I also don't prefer one or the other way, but I was wondering.